setup.py

```diff
@@ -39,6 +39,7 @@
         'taca.subcommands': [
             'storage = taca.storage.cli:storage',
             'analysis = taca.analysis.cli:analysis',
+            'bioinfo = taca.utils.cli:bioinfo',
```

Comment:
@Galithil this subcommands are used for the delivery page, so maybe better use a more intuitive name than 'bioinfo'. What about 'deliveries' or 'delivery-page'